app-text/libepubgen/libepubgen-0.0.0.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="EPUB generator for librevenge"
+DESCRIPTION="libepubgen is an EPUB generator for librevenge. It supports \
+librevenge's text document interface and--currently in a very limited \
+way--presentation and vector drawing interfaces."
+HOMEPAGE="https://sourceforge.net/projects/libepubgen"
+COPYRIGHT="2016 David Tardon"
+LICENSE="MPL v2.0"
+REVISION="1"
+SOURCE_URI="https://heanet.dl.sourceforge.net/project/libepubgen/libepubgen-$portVersion/libepubgen-$portVersion.tar.gz"
```

Comment:
Instead of using a specific mirror you should use the download redirector service.
For that purpose you can use:
```
SOURCE_URI="https://downloads.sourceforge.net/libepubgen/libepubgen-$portVersion.tar.gz"
```
or
```
SOURCE_URI="https://downloads.sf.net/libepubgen/libepubgen-$portVersion.tar.gz"
```